force-app/main/default/classes/RD2_PauseForm_CTRL.cls

```diff
@@ -143,6 +143,29 @@ public with sharing class RD2_PauseForm_CTRL {
             UTIL_Permissions.canUpdate(rdObjectName, false);
     }
 
+    private static Boolean hasFieldReadAccess(Set<String> queryFields) {
+        String nameSpacedField;
+
+        for (String queryField:queryFields) {
```

Comment:
CommitmentId__c is one of the fields being checked for access. I think we should probably remove that field from the access check. What do you think??

Reply:
We need access to it, but I think it falls into the category of something that is system-maintained, not something that needs to be accessed by a user.

Reply:
CommitmentId definitely falls into the System-maintained side and should not be checked for access.

Reply:
On top of `CommitmentId`, I think we should also remove the following fields from FLS check because those aren't related to Pause functionality and it's probably used for querying/updating the schedules. In the UI we only display Installment Date, Amount, and Payment Method.

- npe03__Recurring_Donation_Campaign__r.Name
- npe03__Installments__c
- npe03__Total_Paid_Installments__c
- npe03__Contact__c
- npe03__Organization__c

What do you think

Reply:
That sounds right to me!